src/browser/ui/__tests__/ReactDOMComponent-test.js

```diff
@@ -140,6 +140,51 @@ describe('ReactDOMComponent', function() {
       expect(stubStyle.color).toEqual('green');
     });
 
+    it('should reject haxors on initial markup', function() {
+      var container = document.createElement('div');
+
+      expect(function() {
+        var element = React.createElement(
+          'x-foo-component',
+          {'blah" onclick="beevil" noise="hi': 'selected'},
+          null
+        );
+        React.render(element, container);
+      }).toThrow();
+    });
+
+    it('should reject haxors on update', function() {
+      var container = document.createElement('div');
+
+      var beforeUpdate = React.createElement('x-foo-component', {}, null);
+      React.render(beforeUpdate, container);
+
+      expect(function() {
+        var afterUpdate = React.createElement(
+          'x-foo-component',
+          {'blah" onclick="beevil" noise="hi': 'selected'},
+          null
+        );
+        React.render(afterUpdate, container);
+      }).toThrow();
```

Comment:
Never mind -- looks like this probably throws because there's no INVALID_PROPERTY_ERRORS defined? I prefer including the exception message here so that we make sure the right error is thrown.